OrbitQt/main.cpp

```diff
@@ -193,6 +193,11 @@ static void StyleOrbit(QApplication& app) {
   app.setStyleSheet(
       "QToolTip { color: #ffffff; background-color: #2a82da; border: 1px "
       "solid white; }");
+
+  QFontDatabase::addApplicationFont(
+      Path::JoinPath({Path::GetFontsPath(), "Caveat-Regular.ttf"}).c_str());
```

Comment:
Can you replace std::string::c_str() by QString::fromStdString?

Reply:
Done